pkg/config/stringsource.go

```diff
@@ -60,6 +60,9 @@ func ResolveStringValue(s configv1.StringSource) (string, error) {
 		return "", fmt.Errorf("no valid PEM block of type %q found in key", StringSourceKeyBlockType)
 	}
 
+	// Deprecated: Legacy PEM encryption as specified in RFC 1423 is insecure by
```

Comment:
so what's the alternative?

Reply:
I don't know, as we are just the receiver of that unsecure thingy :smile:

Reply:
Let me add a TODO upfront, so we can tackle it with the TODO story, that I just created.